internal/pkg/agent/application/upgrade/step_download_test.go

```diff
@@ -83,13 +83,17 @@ func TestDownloadWithRetries(t *testing.T) {
 		},
 	}
 
+	CreateMarkerTestDir(t)
```

Comment:
want to do cleanup after?

Reply:
The `CreateMarkerTestDir` does the cleanup.  That's why it's passed the `t`.